source/plugins/commands.rst

```diff
@@ -22,39 +141,74 @@ The first step is to create a class for the command. The class has to implement
 
     import com.google.common.base.Optional;
 
+    // Simple command that sends a message to all players: /broadcast <message>
     public class MyBroadcastCommand implements CommandCallable {
 
         private final Server server;
-        private final Optional<String> desc = Optional.of("Displays a message to all players");
-        private final Optional<String> help = Optional.of("Displays a message to all players. It has no color support!");
+
+        private final Optional<String> desc = Optional.of(
+            "Displays a message to all players");
+
+        private final Optional<String> help = Optional.of(
+            "Displays a message to all players. It has no color support!");
 
         public MyBroadcastCommand(Server server) {
             this.server = server;
         }
 
-        public boolean call(CommandSource source, String arguments, List<String> parents) throws CommandException {
-            server.broadcastMessage(Messages.of(arguments));
+        // Called when a CommandSource (player, console, ...) executes the command
+        public boolean call(CommandSource source, String arguments,
```

Comment:
@boformer Once again, please [read the class](https://github.com/SpongePowered/SpongeAPI/blob/master/src/main/java/org/spongepowered/api/util/command/CommandCallable.java#L44-L45) before saying anything about it.

@saladoc I'm pinging @sk89q in order to resolve this